src/extension.ts

```diff
@@ -204,8 +204,12 @@ export class Extension {
     ];
     await this._rebuildModel(false);
 
-    const fileSystemWatcher = this._vscode.workspace.createFileSystemWatcher('**/*playwright*.config.{ts,js,mjs}');
-    this._disposables.push(fileSystemWatcher);
+    const fileSystemWatchers = [
+      // Glob parser does not supported nested group, hence multiple watchers.
+      this._vscode.workspace.createFileSystemWatcher('**/*playwright*.config.{ts,js,mjs}'),
+      this._vscode.workspace.createFileSystemWatcher('**/.env*'),
```

Comment:
```suggestion
      this._vscode.workspace.createFileSystemWatcher('**/*.env*'),
```

Maybe like that, so it works for https://github.com/microsoft/playwright/issues/28874#issuecomment-1887315841 out of the box.

Reply:
*.env is fine as long as it covers .env